rdr_service/resource/generators/consent_metrics.py

```diff
@@ -312,6 +318,12 @@ def _is_potential_false_positive_for_missing_signature(resource, expected_sign_d
                 row.other_errors.find(ConsentOtherErrors.VETERAN_CONSENT_FOR_NON_VETERAN) != -1
             data['invalid_printed_name'] =\
                 row.other_errors.find(ConsentOtherErrors.INVALID_PRINTED_NAME) != -1
+            data['sensitive_ehr_expected'] =\
+                row.other_errors.find(ConsentOtherErrors.SENSITIVE_EHR_EXPECTED) != -1
+            data['non_sensitive_ehr_expected'] =\
+                row.other_errors.find(ConsentOtherErrors.NONSENSITIVE_EHR_EXPECTED) != -1
+            data['sensitive_ehr_missing_initials'] =\
+                row.other_errors.find(ConsentOtherErrors.INITIALS_MISSING_ON_SENSITIVE_EHR) != -1
```

Comment:
:nit: These checks may be more readable as something like this
```
data['sensitive_ehr_missing_initials'] = ConsentOtherErrors.INITIALS_MISSING_ON_SENSITIVE_EHR in row.other_errors
```

Reply:
noted, will look at this on next update to this method.